prime-router/.java-version

```diff
@@ -1 +1 @@
-11
+17
```

Comment:
```suggestion
17.0
```

Reply:
Did jEnv fail to resolve 11 in your environment? It resolved to the latest point release of 11 for me (jdk-11.0.20.1+1). This may be desirable as we'll get bugfixes from new point releases for free.

Also, should we consider removing this file altogether?
We could also consider sdkman instead, a similar tool that's more common/modern and can manage other SDKs as well.

Reply:
I'm actually not certain about 11 as I don't have openJDK 11 installed on my machine, and did not install it in my testing. This was the output of running `jenv versions` on my machine:
```
jenv: version `17' is not installed
  system
  17.0
  17.0.8
  20.0
  20.0.1
  openjdk64-17.0.8
  openjdk64-20.0.1
```
I have no objections to removing the file altogether.